python/docs/source/user_guide/pandas_on_spark/options.rst

```diff
@@ -208,6 +208,16 @@ This is conceptually equivalent to the PySpark example as below:
     >>> spark_df.rdd.zipWithIndex().map(lambda p: p[1]).collect()
     [0, 1, 2]
 
+.. warning::
+    Unlike `sequence`, since `distributed-sequence` is executed in a distributed environment,
+    the rows corresponding to each index can be different although the index itself is still
```

Comment:
Nit suggestion: "the rows corresponding to each index may vary"

another Nit suggestion: "generated globally sequential" → "remains globally sequential"

Reply:
Thanks @the-sakthi ! Just applied the suggestions

Reply:
Thanks for the quick revision with the suggestions considered, @itholic

Reply:
Thanks for the review, @the-sakthi :)